mypy.ini

```diff
@@ -3,6 +3,6 @@ python_version = 3.7
 ignore_missing_imports = True
 files = cylc/uiserver
 # don't run mypy on these files directly
-exclude = cylc/uiserver/(websockets/tornado.py|config_defaults.py)
+exclude = cylc/uiserver/(websockets/tornado.py|config_defaults.py|tests/*)
```

Comment:
MyPy really doesn't like the compiled protobuf file imported in the tests. Inline ignore comments didn't seem to be enough to suppress the error. Tried copying the cylc-flow mypy config but that didn't fix it so just ignoring the tests for now, can re-instate later.

Reply:
Did you add the `# type: ignore` in the ... wrong line? I noticed from @MetRonnie that he had to add the comment in the initial line when you had a multi-line import.

If I remove `test/*` from the exclusion in your branch, the command fails. Then this diff fixes it.

```diff
diff --git a/cylc/uiserver/tests/conftest.py b/cylc/uiserver/tests/conftest.py
index 167ba6c..a8a838e 100644
--- a/cylc/uiserver/tests/conftest.py
+++ b/cylc/uiserver/tests/conftest.py
@@ -19,7 +19,7 @@ import inspect
 
 import pytest
 import zmq
-from cylc.flow.data_messages_pb2 import (
+from cylc.flow.data_messages_pb2 import (  # type: ignore
     PbEntireWorkflow, PbWorkflow, PbFamilyProxy)
 from cylc.flow.network import ZMQSocketBase
 
diff --git a/mypy.ini b/mypy.ini
index 20eca2f..b0b4d93 100644
--- a/mypy.ini
+++ b/mypy.ini
@@ -3,6 +3,6 @@ python_version = 3.7
 ignore_missing_imports = True
 files = cylc/uiserver
 # don't run mypy on these files directly
-exclude = cylc/uiserver/(websockets/tornado.py|config_defaults.py|tests/*)
+exclude = cylc/uiserver/(websockets/tornado.py|config_defaults.py)
 # don't run mypy on those files when imported from other modules
 follow_imports = silent

```

Reply:
Yes, it only worked for me when I used `# type: ignore` on the first line of the multiline import

Reply:
Tried `type: ignore` but there were problems besides that, managed to get them fixed here https://github.com/cylc/cylc-uiserver/pull/188/commits/0029244360b68328861211bb8ff24ab4a5ab5f8f